modules/replication/dockerimage-replication/modulestack.yaml

```diff
@@ -35,7 +35,9 @@ Resources:
             Resource:
               - !Sub "arn:aws:ecr:${AWS::Region}:${AWS::AccountId}:repository/addf-*"
           - Action:
-              - "ssm:PutParameter"
+              - "s3:CreateBucket"
+              - "s3:ListBucket"
+              - "s3:PutObject"
```

Comment:
This give you permissions to write to any S3 bucket....that sounds bad.  Can this be restricted to a bucket?

Reply:
Done